content/en/os/1.13.x/update/guidelines.markdown

```diff
@@ -0,0 +1,62 @@
+# Guidelines for Updating In-Place
+
+## Introduction
+
+There are some guidelines and considerations to keep in mind when updating your Bottlerocket nodes _in-place_.
+Some movements between versions are updates, some are not updates, and there are some caveats you may need to consider based on the history of the project.
+
+## Update Paths
+
+Certain movements between releases of Bottlerocket are accessible through an _in-place_ update path.
+
+### Between Minor or Patch Versions of the Same Variant
+
+With the exception of major versions, it is possible to update from one released version of a variant to another released version of the same variant.
+In particular, the following update paths work:
+
+- Minor version to minor version (e.g. `1.10.0` to `1.11.0`)
+- Patch version to patch version (e.g. `1.9.1` to `1.9.2`)
+- Minor version to patch version (e.g. `1.10.0` to `1.10.1`)
+- Patch version to minor version (e.g. `1.10.1` to `1.11.0`)
+
+Furthermore, skipping minor or patch versions of the same variant is allowed.
+There is a [caveat discussed below](#too-many-releases-between-versions), however, if there are too many minor or patch releases between your initial and target version.
```

Comment:
This basically restates the section you are linking.

Reply:
Moved this to the intro sentence as suggested in https://github.com/bottlerocket-os/project-website/pull/32#discussion_r1113590233